apps/meteor/app/lib/server/functions/createRoom.ts

```diff
@@ -136,7 +136,7 @@ export const createRoom = function <T extends RoomType>(
 			try {
 				callbacks.run('federation.beforeAddUserAToRoom', { user: member, inviter: owner }, room);
 			} catch (error) {
-				throw new Meteor.Error((error as any)?.message);
+				continue;
```

Comment:
The original idea was to behave like that, but this was leading to strange behavior since the room was created, but with that throw, the subscriptions were not being created, even for the room creator. So I've changed this to allow the subscription creation for the users who exist (in this case, the creator is 100% sure that exist), and I handle the federation stuff in the other callback.